packages/components/src/components/infotext/infotext.scss

```
@@ -59,9 +26,9 @@ $variants: (
 		}
 	}
 
-	@each $name, $colors in $variants {
+	@each $name, $colors in $component-variants {
 		&[data-variant="#{$name}"] {
-			color: nth($colors, 1);
+			color: nth($colors, 4);
```

what about using an keys for the values within the `$component-variants` lowest level? Than you would be able to more clearly select for a specific entry in here by it's name instead of its index with [`map.get`](https://sass-lang.com/documentation/modules/map#get):

```suggestion
			color: map.get($colors, "successful");
```

or `map.get($colors, "border");`, `map.get($colors, "element");` or `map.get($colors, "on-bg");` for the other three entries.

---

Yes, it's even better, I changed it.